frontend/src/pages/Roster.js

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import MaterialTable from "@material-table/core";
+import { Helmet } from "react-helmet";
+import { Grid, Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { useDispatch } from "react-redux";
+import { TableIcons } from "../components/Icons";
+import LoadingContainer from "../components/LoadingContainer";
+import PageContainer from "../components/PageContainer";
+import { openAlert } from "../actions";
+import { getUsers } from "../services/users";
+import { withAuthorization } from "../components/HOC";
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    textAlign: "center",
+  },
+  dates: {
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const styleButton = {
+  color: "inherit",
+  fontWeight: "900",
+  textDecoration: "underline",
+  textTransform: "none",
+  background: "transparent",
+};
+const deafultButton = {
```

Comment:
```suggestion
const defaultButton = {
```
Just a spelling issue.